electron/src/electron/network.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  WebPlugin, NetworkPlugin, NetworkStatus
+} from "@capacitor/core";
+
+export interface PluginListenerHandle {
+  remove: () => void;
+}
+
+declare var window:any;
+
+export class NetworkPluginElectron extends WebPlugin implements NetworkPlugin {
+
+  listenerFunction:any = null;
+
+  constructor() {
+    super({
+      name: 'Network',
+      platforms: ['electron']
+    });
+  }
+
+  getStatus(): Promise<NetworkStatus> {
+    return new Promise((resolve, reject) => {
+      if(!window.navigator)
```

Comment:
Try to avoid `if` and `else` without curlies. Can you change this to `if (!window.navigator) {`

Reply:
Okay, will fix